packages/core/datadog-configuration.schema.json

```
@@ -194,7 +235,11 @@
                     "type": "boolean"
                 }
             },
-            "required": ["clientToken", "env", "applicationId"]
+            "required": [
+                "clientToken",
+                "env",
+                "applicationId"
+            ]
         }
     }
-}
+}
```

should we add a formatter for JSON in this project? seems nothing adds newline.

```suggestion
}

```